### PR TITLE
srm landing page

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: src/components/Docs/data/serviceReliabilityManagementData.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      time: "9min",
+      link: "/tutorials/build-code/build/tfc-notification",
+    },
+  ];
+  */
+  
+  // Docs
+  export const docsCards: CardSections = [
+    {
+      name: "Essentials",
```
Comment (KimberlyFields) [inline src/components/Docs/data/serviceReliabilityManagementData.ts:39]: Replace this tile title with "Get started".

File: src/components/Docs/data/serviceReliabilityManagementData.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+          module: MODULES.srm,
+          description:
+            "Understand concepts such as SLOs, SLIs, and error budgets.",
+          link: "/docs/service-reliability-management/getting-started/service-reliability-management-basics",
+        },
+        {
+          title: "Your first SLO",
+          module: MODULES.srm,
+          description:
+            "Learn how to create, track, and monitor a SLO.",
```
Comment (KimberlyFields) [inline src/components/Docs/data/serviceReliabilityManagementData.ts:61]:
```suggestion
            "Learn how to create, track, and monitor an SLO.",
```

Comment (SudheendraKatte) [reply src/components/Docs/data/serviceReliabilityManagementData.ts]: Updated

File: src/components/Docs/data/serviceReliabilityManagementData.ts [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  
+  // Docs
+  export const docsCards: CardSections = [
+    {
+      name: "Essentials",
+      description:
+        "",
+      list: [
+        {
+          title: "Overview",
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply src/components/Docs/data/serviceReliabilityManagementData.ts:44]: Updated

File: src/components/Docs/data/serviceReliabilityManagementData.ts (hunk 1 above)
Comment (SudheendraKatte) [reply src/components/Docs/data/serviceReliabilityManagementData.ts]: Updated

**Comment by SudheendraKatte** [comment]
Already reviewed by Kim.

**Pull request merged**